Source/Core/InputCommon/ControllerEmu/ControlGroup/Force.cpp

```diff
@@ -26,26 +25,69 @@ Force::Force(const std::string& name_) : ControlGroup(name_, GroupType::Force)
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Forward")));
   controls.emplace_back(std::make_unique<Input>(Translate, _trans("Backward")));
 
-  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Dead Zone"), 0, 0, 50));
+  // Maximum swing movement (centimeters).
+  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Distance"), 0.25, 1, 100));
+
+  // Maximum jerk (m/s^3).
+  // i18n: "Jerk" as it relates to physics. The time derivative of acceleration.
+  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Jerk"), 5.0, 1, 1000));
+
+  // Angle of twist applied at the extremities of the swing (degrees).
+  numeric_settings.emplace_back(std::make_unique<NumericSetting>(_trans("Angle"), 0.45, 0, 180));
 }
 
-Force::StateData Force::GetState()
+Force::ReshapeData Force::GetReshapableState(bool adjusted)
 {
-  StateData state_data;
-  const ControlState deadzone = numeric_settings[0]->GetValue();
+  const ControlState y = controls[0]->control_ref->State() - controls[1]->control_ref->State();
+  const ControlState x = controls[3]->control_ref->State() - controls[2]->control_ref->State();
```

Comment:
0 - 1 - 3 - 2 looks a bit strange; I hope thats correct.

Reply:
Yeah.. It's correct. Just some ugly sign flips.. 0,1,2,3 are Up,Down,Left,Right mappings and positive X is to the right but positive Y is up. It could use some cleanup but I'll save that for another PR.